### ISSUE TITLE
Improve Expired Account Verification Token Message

### ISSUE THREAD

**Issue by lautarodragan** [poetapp/frost-web#47]
When a user follows the account verification link they receive in their email, if that link is expired, they'll see a message like this:

```
Server failed to authenticate the request. Make sure the value of the Authorization header is formed correctly including the signature
```

A few people have approached me asking what they should do since they were receiving this error message.

We need to improve this message, stating exactly what's going on and steps to solve the issue.

Something in the lines of

```
Sorry! Looks like that link expired. Account verification links expire after X hours for security purposes. 

Please request a new one by logging in, going to the API Tokens screen and clicking on the <Verify Account> button. You'll receive a new account verification email.
```

**Comment by jwicks31** [comment]
While I was trying to reproduce this issue I noticed that there are several different reasons that we would be getting this error, however it doesn't seem to be because of an expired token. 

This error is thrown inside of authorization.ts middleware in the frost-api. I wasn't able to consistently reproduce the error message above, but it would sometimes be caused by the failing `Vault.readSecret` or `Vault.verifyToken`.  Every-time I saw this error it was due to `Error: read ECONNRESET` falling through and displaying the default error message. 

There were times whenever this error would throw when `jsonwebtoken.verify` was failing as well, but it would the error messages were about incorrect JSON. Other times this would correctly fail with the message 'bad token' and display the "Invalid Token" error message. I wasn't able to get any consistency here either. 

I am creating a PR that changes the wording for whenever the token is expired. The message is what you would get when trying to use an expired link, whether or not you have a verified account. So, not sure if we will want to reword it someway. Right now it reads:
`Sorry! Looks like that link expired. Please login to the API Keys screen and click on the <Resend Email> button. You'll receive a new account verification email.`

**Comment by lautarodragan** [comment]
We should be careful to display different error messages depending on
- whether the user is making a call to the API to create a Work using an expired token, or
- whether the user is making a call to the API to verify the email account (i.e., clicked on the mail's link) with an expired Verify Account token

There's different types of tokens and we need to display different messages for the different cases.

@wzalazar can you sync with @jwicks31 on this? I know it could be kind of tricky to set up.

**Comment by jwicks31** [comment]
Was this issue resolved with #81? I believe that the changes @lautarodragan is talking about above would need to be made in Frost-api. Should we create an issue there?
